lib/templates/mixins/seo/getMeta.js

```diff
@@ -1,13 +1,21 @@
 export default {
   methods: {
     getMeta() {
-      const pageData = this.page.page
       const baseUrl = this.$typo3.options.baseURL
-      const meta = {
-        title:
-          pageData.meta.title ||
-          pageData.title ||
-          this.$nuxt.$options.head.title,
+      const { meta, socialMedia, title } = this.page.page
+      const { canonical, robots } = meta
```

Comment:
`robots` is available in TYPO3 headless ext >= 2.5. We should add fallback here to check if robots exist to keep backward compatibility. You can test it with https://api.t3pwa.com/ - now we have old version of headless ext here so `robots` field is not available.

Reply:
`robots` from `const { canonical, robots } = meta ` will result as undefined.

the simple solution would be to write `Object.keys(robots || {}) ` on line 36.

Reply:
I think it will be enough 👍